src/index.ts

```diff
@@ -0,0 +1,145 @@
+
+  walletmonitor!.forEach((item) => {
+    const { address } = item;
+    wallets.push(address);
+
+    logger.info(`Subscribing to account changes for ${address}`);
+    connection.onAccountChange(
+      new PublicKey(address),
+      onAccountChangeCallBack,
+      "confirmed"
+    );
+  });
+};
+
+const onAccountChangeCallBack = async (
+  accountInfo: AccountInfo<Buffer>,
+  context: Context
+) => {
+  logger.info("Account change detected");
+  const { slot } = context;
+  let wallet: PublicKey | null = null;
+
+  try {
+    logger.info(`Retrieving block in slot: ${slot}`);
+    let block: BlockResponse = (await connection.getBlock(
+      slot
+    )) as BlockResponse;
+    const { transactions } = block;
+
+    logger.info("Searching transactions");
+    const transaction = transactions.find((item) => {
+      const { accountKeys } = item.transaction.message;
+      let publicKey = accountKeys.find((publicKey) =>
+        wallets.includes(publicKey.toString())
+      );
+      if (publicKey) {
+        wallet = publicKey;
+        return item;
+      }
+    })?.transaction;
+
+    logger.info(`Transaction found: ${transaction}`);
+    const signature: string = transaction!.signatures[0] as string;
+    logger.info(`Getting transaction signature: ${signature}`);
+
+    const txn: TransactionResponse = (await connection.getTransaction(
+      signature
+    )) as TransactionResponse;
+    const { preBalances, postBalances, postTokenBalances, preTokenBalances } =
+      txn.meta as ConfirmedTransactionMeta;
+    const price = Math.abs(preBalances[0] - postBalances[0]) / LAMPORTS_PER_SOL;
+    let mintToken: string = postTokenBalances![0]?.mint;
+
+    if (mintToken) {
+      let tradeDirection = "";
+      const { accountKeys } = txn.transaction.message;
+      const programAccount: string = accountKeys.at(-1)!.toString();
+
+      for (const [key, value] of Object.entries(PROGRAM_ACCOUNTS)) {
+        if (value.includes(programAccount)) {
+          let programAccountUrl = PROGRAM_ACCOUNT_URLS[key];
+          let walletString = wallet!.toString();
+
+          if (key === "MortuaryInc") {
+            tradeDirection = BURN;
+            mintToken = preTokenBalances![1].mint;
+          } else if (price < 0.009) {
+            tradeDirection =
+              preTokenBalances![0].owner === walletString
+                ? LISTING
+                : DE_LISTING;
+          } else if (key === "Magic Eden") {
+            programAccountUrl += `/${mintToken}`;
+            tradeDirection =
+              preTokenBalances![0].owner === walletString ? SELL : BUY;
+          } else {
+            programAccountUrl += `/?token=${mintToken}`;
+            tradeDirection =
+              postTokenBalances![0].owner === walletString ? BUY : SELL;
+          }
+
+          const metadata = await getMetaData(mintToken);
+          const nftMeta: NFTMetaType = {
+            name: metadata.name,
+            tradeDirection: tradeDirection,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Stonks-Luma-Liberty/GuiltySpark/pullRequest?prid=8929681)
```suggestion
            tradeDirection,
```